photon-core/src/main/java/org/photonvision/vision/pipeline/ArucoPipeline.java

```diff
@@ -85,19 +85,25 @@ protected void setPipeParamsImpl() {
         // 2023/other: best guess is 6in
         double tagWidth = Units.inchesToMeters(6);
         TargetModel tagModel = TargetModel.kAprilTag16h5;
-        switch (settings.tagFamily) {
-            case kTag36h11:
-                // 2024 tag, 6.5in
-                params.tagFamily = Objdetect.DICT_APRILTAG_36h11;
-                tagWidth = Units.inchesToMeters(6.5);
-                tagModel = TargetModel.kAprilTag36h11;
-                break;
-            case kTag25h9:
-                params.tagFamily = Objdetect.DICT_APRILTAG_25h9;
-                break;
-            default:
-                params.tagFamily = Objdetect.DICT_APRILTAG_16h5;
-        }
+
+        params.tagFamily =
+                switch (settings.tagFamily) {
+                    case kTag36h11 -> {
+                        // 2024 tag, 6.5in
+                        tagWidth = Units.inchesToMeters(6.5);
+                        tagModel = TargetModel.kAprilTag36h11;
+                        yield Objdetect.DICT_APRILTAG_36h11;
+                    }
+                    case kTag25h9 -> Objdetect.DICT_APRILTAG_25h9;
+                        // TODO: Somebody please approve that this is the desired behavior
+                    case kTag16h5,
+                                    kTagCircle21h7,
+                                    kTagCircle49h12,
+                                    kTagCustom48h11,
+                                    kTagStandard41h12,
+                                    kTagStandard52h13 ->
+                            Objdetect.DICT_APRILTAG_16h5;
```

Comment:
ditto

Reply:
afaik nobody's even tried testing kTag16h5,
                                    kTagCircle21h7,
                                    kTagCircle49h12,
                                    kTagCustom48h11,
                                    kTagStandard41h12, or
                                    kTagStandard52h13 for Aruco. Maybe we should more expicitly drop support for these non-standard weird dicts instead entirely.

Reply:
Aight I'll change this to default to 36h11 as thats the default we are using rn and open an issue.